Tools/Scripts/set-webkit-configuration

```diff
@@ -44,6 +44,7 @@ Usage: $programName [options]
   --[no-]asan                        Enable or disable clang address sanitizer
   --[no-]tsan                        Enable or disable clang thread sanitizer
   --[no-]ubsan                       Enable or disable clang undefined behavior sanitizer
+  --[no-]libfuzzer                   Enable or disable clang libfuzzer
```

Comment:
The name of the library is `libFuzzer`, so my OCD really wants just the description fixed (the `--libfuzzer` switch can remain all lowercase).

However, I don't think this needs to be fixed before landing the PR.

Reply:
Oh really? I thought it was “libfuzzer”

Reply:
You're right! https://www.llvm.org/docs/LibFuzzer.html